src/configs/price.ts

```diff
@@ -8,27 +8,22 @@ export const DefaultPriceConfig: PriceConfig = {
     {
       name: "Time: <1 Hour",
       weight: 0.125,
```

Comment:
Pretty sure this refers to days. There's 8 working hours in a day. 
Perhaps as part of this implementation we can batch every six hours as a day (I believe that most people only put in six productive hours in a day)

Reply:
I feel this looks good. 8 parts a day. doing 6 makes the number go longs. Here comes rounding of number.

Reply:
Can you change to 

`1/6=0.166666666666667`